nhrpd/nhrp_multicast.c

```diff
@@ -47,19 +47,22 @@ static void nhrp_multicast_send(struct nhrp_peer *p, struct zbuf *zb)
 
 	addrlen = sockunion_get_addrlen(&p->vc->remote.nbma);
 	ret = os_sendmsg(zb->head, zbuf_used(zb), p->ifp->ifindex,
-		   sockunion_get_addr(&p->vc->remote.nbma),
-		   addrlen, addrlen == 4 ? 0x0800 : 0x86DD);
-
-	debugf(NHRP_DEBUG_COMMON, "Multicast Packet: %s -> %s, ret = %d, size = %zu, addrlen = %zu",
-		   sockunion2str(&p->vc->local.nbma, buf[0], sizeof(buf[0])),
-		   sockunion2str(&p->vc->remote.nbma, buf[1], sizeof(buf[1])),
-		   ret, zbuf_used(zb), addrlen);
+			 sockunion_get_addr(&p->vc->remote.nbma), addrlen,
+			 addrlen == 4 ? 0x0800 : 0x86DD);
+
```

Comment:
it should be the good time to add if_ether.h in 
https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/if_ether.h#L54, in include/linux/if_ether.h

Alternatively, just add on the header definition of ETH_P_IPV6 ( I see that ETH_ALEN is also redefined in prefix.h, and adding if_ether.h may be more complicated than expected)

Reply:
linux/if_ether.h is already included via netinet/if_ether.h. So very simple to change code to use defines instead of hard-coded values.